### PR TITLE
0022583: ilMimeTypeUtil: Some minor issues

### PR DESCRIPTION
**Pull request by bseglias**
https://www.ilias.de/mantis/view.php?id=22583

### CODE REVIEW

File: Services/Utilities/classes/class.ilMimeTypeUtil.php [hunk 1]
```diff
@@ -1445,7 +1445,7 @@ class ilMimeTypeUtil {
 	 * @param $path_to_file
 	 */
 	protected function __construct($path_to_file) {
-		if (strpos($path_to_file, 'http://') || strpos($path_to_file, 'https://')) {
+		if (strpos($path_to_file, 'http://') !== FALSE || strpos($path_to_file, 'https://') !== FALSE) {
```
Comment (chfsx) [inline Services/Utilities/classes/class.ilMimeTypeUtil.php]: Hi @bseglias could you change `FALSE` to lowercase `false`? Thanks!

**Review by chfsx — CHANGES_REQUESTED** [review]

**Comment by chfsx** [comment]
Hi @alex40724 LGTM, are these changes for you OK as well?

**Review by chfsx — APPROVED** [review]

**Comment by alex40724** [comment]
The bug report does not state how to reproduce any bug. What exactly do I need to do? What behaviour is fixed? The code changes look good, but if there is no bug, I would only put this into the trunk as code improvements.

**Comment by chfsx** [comment]
@mjansenDatabay would it be possible for you to add a sentence about Reproducibility in the mantis ticket? Thanks in advance!

**Comment by mjansenDatabay** [comment]
@chfsx What do you mean by 'Reproducibility'? There is no current issue with the ILIAS core code base. I noticed the issues just by reviewing the existing code, because I have to rely on \ilObjMediaObject::getMimeType in a plugin (which I cannot ATM).

**Comment by chfsx** [comment]
@mjansenDatabay for my part of the PR this is fine, many thanks! For me it's definitely a bug when writing "wrong" code or code which may break. there is no need fo a Fatal or something like that. The current code potentially leads to problem and you already found one using your plugin.

@alex40724 it's up to you whether you accept your party or not, feel free to give a feedback accordingly. Benjamin will adjust the PR then.

**Comment by alex40724** [comment]
Personally I tend to not accept bugs/code changes in stable releases, if I cannot reproduce a bug. Just reading the code is not a proper quality check.

@mjansenDatabay: Please at least write a short description what kind of bug you get in your plugin and that this change fixes this bug in the bug report. Then it is ok for me.

**Review by alex40724 — APPROVED** [review]

**Comment by mjansenDatabay** [comment]
@alex40724 The bug is: If you pass a string like 'https://www.youtube.com/watch?v=WSgP85kr6eU', you will get 'application/octet-stream' (the passed fallback mime type). If this PR (https://github.com/ILIAS-eLearning/ILIAS/pull/892) is merged, you will get 'video/youtube'.

I added some tests, see: https://github.com/ILIAS-eLearning/ILIAS/pull/944

**Comment by alex40724** [comment]
Great, thanks. I merged this to 5.3/trunk.

**Pull request merged**